elastic/integrations
# TSDB Enablement for InfluxDB

- Enhancement

## What does this PR do?
- TSDB Enablement for InfluxDB
- Minor dashboard fix



The change includes 
- adding new fields that are essential for dimension mapping and field mapping for the newly added field
- tag fields that are dimensions


## Checklist


## Author's Checklist



## How to test this PR locally

- Download, Install, configure Influxdb
- Download, Install, configure telegraf
- Configure telegraf to inject data into the influxdb
- Create tasks in influxdb to generate task related metrics
- Confiugure the integration
- Verify the dashboard is loaded correctly without missing data except panels reporting errors. 



## Related issues

https://github.com/elastic/elastic-package/issues/1174


-

## Screenshots
